.github/label-actions.yml

```diff
@@ -123,3 +123,11 @@ issues:
         When creating an issue, please ensure that the default issue template has been updated with the required details.
 
         Closing this issue. If you believe this issue has been closed in error, please provide any relevant output and logs which may be useful in diagnosing the issue.
+
+    ruby-3.0.0:
+      close: true
+      comment: |
+        This issue appears to be related to Ruby 3.0.0. At this time Metasploit does not support Ruby 3.0.0.
+        Please try using Ruby 2.7.x with Metasploit.
+
+        Closing this issue as a duplicate of #14666 - which aims to track this feature request.
```

Comment:
This should take the user to https://github.com/rapid7/metasploit-framework/issues/14666